examples/notebooks/introduction_enlyze_python_sdk.ipynb

```diff
@@ -0,0 +1,2981 @@
Hi8mddXfBBN+jEeLQ5AWhOpnTM/t01a1fn2w6uJpFSOrA/w44/OZ3qZN1O8fv/ek6IoUOZz0h5K0BVOUl2YL2b4iktYcOqZe3RMb1HxbkWBgw1RkigoK8+9H/56JA63m3knnbrL47n9BfusbfsLb3ExetwcAwlGfhROrubIGAXZVYaWlWaM7YP2Ev7n4dWs2NFPeyYllm6AnhFGqgtd/PmLR9NQ7N5tFykeYv2QpmmskvtYtKhmt/3IRxZV8+GFlegGnDWvAkEQ0HWDl7f20dwf50+fX5pL3zIeibSG1ichGTqyoeaJI9nvqyXuIGXXaSh28/UPTqc1GCeSVPMMmgA1ATfzqvzMqfLjsU/MtHEgnlvvZzq2pHngMavmyOFOdj6REJ2EEwp/eaOFv2SMyj6nTMC969RHumE6NDhkEb/Tlhu/64aZ4shlk/A5ZboHU9gkgYoCJ7JxFJ+1XUCBEaY77eT19GLa2mbxIbmRCqMHydCIGE4SioaTFIOGh9dLzuexc+Yc9M/lvakrN9Eo3Onlex4Buz8w5yR9LFpUf1DPST40v5K0pnPMlGJskpBL8wXQGU4iCQJxRSWSVJla6sXnlNF0U5TqCCVo6otz3NRi9EwKLJv03mqXhRMKHeEE9cUe7JJoCniKxroXbspt0xc1xRpVN7j9he1ceHQdsyt3EdVlsccc0uKILWPAisfjuFzvzSvMwsLi/Y9hGKQzE7CGgBunY+xHYFrViCtmvk1ZEnHIInJmwGVk/pedmumGgSwK2OX3PgAQhqobIGbzQSOanq+qSmIwTF93F9FgP28+dB9aOs2bj/6NsoZGdF0nGY2OMqAPx+5yUVo/BeF9EoliGAbRaJSt//JO6iQ2PhhmoL0VGJ22ZTiCIFI3fyEOl9vcLxKmbcM6AMobp+Nwu+nYsglREikoq8BfWk4yGqF903pcPj81s+flNDIBAVVJ07J2dZ7YFaisoqi6lvbNG0lmJi2BymrSiTixUJBEJqrEEyjGV1SUy8ufRdcy3jNFxVROn4mmKAx0tBHqMr1Aq2bOwVNQOOp6i5mUK3qmCHvXYJIX21WU8mms+NrxuRDkrJdsNoIk59E64n020kTMLBi5XMh4Eu2LYs4WFha7Jhh8HQADAU8mciSFzNef6ECnAwMYTCi53Nt3ffZISrwTLzS+L6goMMWRZzcOPc/G9aQfsV7TdLYNKIhtYSTp/dH/7Qu8Dplo6sB6Pu5ropIHLTNmGVlzRNPN92Ol1VKMYeKIYkaOvNIl8Y+4mRJSEgWuOn0m5y2uAsz+qdhrx5b5fVh28/GRRIEz51XmLRNFgYcuP5Z4WqWyYPdz7uqAC02QAAXZ0PKis7ORI/26aey76aPzWTatZF8132IC2DOOVDY9TXgfiSO6rqEpCqGuTuwuN4IoEA/v/zR2E8YwCPd2mxHVxaUgCKRiMba++RoufwHJ6CCibEP3F9Hl8yAKAl07ttLf2kxkoB8MA1VR0DWVdCKBN1BEKh4nPhjGV1SEkkoh2+yUNU4j2t9HMhZl+tLjCFRW0d20Hbe/wExVK0A8HGbTqy8RD4eonjWHYGcHZQ2NVM2cQ/Oad2ndsAZRFJl61DFIso0tb7yaG5+PRUF5Bf6SMnqbd1A2ZSouXwGDfT2oqRRqOkW4x/zcDreH+gWLGWhvpbtpW24OE5M9nDannBKvgw0dYTZ2RYgk1aF6DbshrepjbhtPa7nIJEUzaB0wxx83cm7+himV2zhp3OP//sx5B70wYiPF1s8AAFFFSURBVHFwsas+ZTxxPzslnVbmY1rZvhWjAh47Ac+Q2GiXRXDICMOiYJPKkGPA/SuauX9FMyfMKGVqqYeucJJYWuPseRVcsLRun7btcOKQFkckSaKwsJCeHjME1+3ei/A/C4v9hGEYpFIpRFG0fpcHiGRaRVPTpugh2NHVsXOBy4A/Fxygg64zXtpwEbMWXVLd+3yUqXQKIy2io6JqOkY6BXqapGoG8hqxGAOd7aQVhf6BIH1bN1E/byHhnm7621ro2r41dyxBECmb0pgzZmeJ9vcT6e+lfdOGvW7nZHEwTJ0EUeToj19I3dwFSJJ5v5rF7sAly4gieItKzBRVaS2XZr2vsxuXZBCorCKt6ZlIRgG7JJoFUJMqRIO4/QVEVIFCtw1RMMOgNd2guz9MuKsdlyzgcjoorW9AMwRURSHW142h65TU1DEQSRBq24nbIVNcW4+CnAnZVvFIOolwCF9xCX0d7UgYFFTWkNaHRAhZgEgoREFREYJghva6bBKKppNUNArddqIp02PT77Txo8c38GqyiU/NLKeiwIpGsrA4VMiKIz2Gm9ebBkatv/Ej8w64MAJmqp63m4Pc8eJ27nhx+94d5LnX922jLPY706N9nFhhWiRGptXSMmLJWGm1spEjoqFRkqk50m0UAvDD8+Zx2myr79rXFLhsFLhGR9aORXWhKyd6SWi5+jgwTByhgNU/OH3Cx7TYd9gyqQaHF2Q3DIN4OESkr5fO7VuI9PUi2ewUVdcQ6esl2NmBpzBApK8HfcSkKdLfR/vmDfssanmyWTWBbXqahvqpYEdb7nXrhrW5181r3t3tcVrWmXURNr360hjr1kygJdC2cV3u9baVK3a57ZYVr+S9Twsy2wrn8O5nj8zZKxRNZ1tPdEIp10q8dlTdIJJU0YalKyzx2okkVVKqGR0WT2v0RFJ5+3odMqpupsASM2mYhIxDVZHHTjhhpqWaWjp+ymMLi/c1w9Jq6QjUFbk5qj7Azv4Y77SEWL6ll+Vbhmq0vL1zgE8eVWuJhXvJIS2OAFRUVADkBBILi4MFwzBQFAWbzWaJIweIWFIhmDCnzLbEwRNNllBT9KkCsqFhS4VJONM49DSkBzAEGaVfJREZRFdVPE4n53zpq4iiiKaqtK5fg5pOI8kyoiQTqKrGX1I66hyGrtO6YS3J6N4VR5sMdF2naedOpjQ0II5R1PVAoekGT3XApauipN5qwiGLeB1yrkCdyybhs
```

Comment:
```suggestion
    "To avoid errors or false data representations, some Python data types need to be converted to supported types. Excel and Power BI don't support timestamps with time zone information and timedeltas. You can use the `DataFrame.info()` method, to see which data types your DataFrame contains."
```